Bottom: d302fa38e05a4a1699d6594f0ece742f57ba8f57
Top:    f90bc7191399665146cfd961f648ce0387789177
Author: Ben Greear <greearb@candelatech.com>
Date:   2014-06-26 11:32:47 -0700

wireless: Allow dynamic interface combinations.

Depending on firmware and/or module parameters, the
number of vdevs supported may change.  Remove const
modifier so that code can change the values accordingly
at run-time.

Signed-off-by: Ben Greear <greearb@candelatech.com>


---

diff --git a/include/net/cfg80211.h b/include/net/cfg80211.h
index 6392167..3ae3030 100644
--- a/include/net/cfg80211.h
+++ b/include/net/cfg80211.h
@@ -2948,7 +2948,7 @@ struct ieee80211_iface_limit {
  *  };
  */
 struct ieee80211_iface_combination {
-	const struct ieee80211_iface_limit *limits;
+	struct ieee80211_iface_limit *limits;
 	u32 num_different_channels;
 	u16 max_interfaces;
 	u8 n_limits;
